mlrun/runtimes/sparkjob/spark3job.py

```diff
@@ -158,15 +164,25 @@ def __init__(
         self.executor_java_options = executor_java_options
         self.driver_cores = driver_cores
         self.executor_cores = executor_cores
+        self.driver_security_context = (
+            driver_security_context
+            or mlrun.mlconf.get_default_function_security_context()
+        )
+        self.executor_security_context = (
+            executor_security_context
+            or mlrun.mlconf.get_default_function_security_context()
+        )
 
     def to_dict(self, fields=None, exclude=None):
         struct = super().to_dict(
             fields,
             exclude=[
                 "executor_affinity",
                 "executor_tolerations",
+                "executor_security_context",
                 "driver_affinity",
                 "driver_tolerations",
+                "driver_security_context",
             ],
```

Comment:
seems like we are overriding the `exclude` param, fix that as you did in `to_dict` (`pod.py`)